NAMESPACE

```diff
@@ -1,6 +1,7 @@
 # Generated by roxygen2: do not edit by hand
 
 export(TAXONOMY_RANKS)
```

Comment:
@FelixErnst ok, that's nice. I haven't reviewed before so I hope this test comment does not bother